catalog.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Техномарт</title>
+  </head>
+  <body>
+
+    <header>
+        <a>
+          <img src="https://via.placeholder.com/350x65" width="150" alt="логотип Техномарт">
+        </a>
+        <input type="search" placeholder="Поиск:">
+        <a href="">Закладки:</a>
+        <a href="">Корзина:</a>
+        <a href="">Оформить заказ</a>
+
+       <h1>
+        <i>Интернет-магазин<br>строительных материалов<br>и инструментов для ремонта</i>
+       </h1>
+      <a href="tel:+78125550555">+7(812)555-05-55</a>
+      <p>г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</p>
```

Comment:
Здесь нужно обернуть в address
```suggestion
      <address>
      <a href="tel:+78125550555">+7(812)555-05-55</a>
      <p>г. Санкт-Петербург, ул. Б. Конюшенная, д. 19/8</p>
      </address>
```